src/api.js

```diff
@@ -11,3 +11,9 @@ export const sendHeartbeat = async (user) => {
   }
   return await fetch('/heartbeat', requestOptions)
 }
+
+export const getStartedGame = async (playerId) => {
+  const response = await fetch(`/my-started-game?playerId=${playerId}`)
```

Comment:
This seems to be okay in the frontend, but calling the endpoint in the backend returns a 500:
```
Error while getting started games 😬 TypeError: Cannot read properties of undefined (reading 'Items')
    at Dynamo.getStartedGames (file:///Users/slavi/git/tic-tac-toe/tic-tac-toe-backend/src/dynamo.js:72:32)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async file:///Users/slavi/git/tic-tac-toe/tic-tac-toe-backend/src/app.js:61:26
```
Does this happen on your end as well?

Reply:
Also what happens when our endpoint returns 404?

Reply:
We don't get this error and also when our endpoint returns 404, it sends 'Not found started games 😬'.